ReactAndroid/src/main/java/com/facebook/react/views/checkbox/ReactCheckBoxManager.java

```diff
@@ -60,4 +65,76 @@ public void setOn(ReactCheckBox view, boolean on) {
     view.setOn(on);
     view.setOnCheckedChangeListener(ON_CHECKED_CHANGE_LISTENER);
   }
+
+  @ReactProp(name = "checkedColor", customType = "Color")
+  public void setCheckedTintColor(ReactCheckBox view, Integer color) {
+    String defaultColorIdOfUncheckedState = "colorPrimaryDark";
```

Comment:
Can we move this to `setCheckboxTintList` instead of passing it as arguments, seems like we always pass the same values.

Reply:
we always pass the same values, but in different order: 

`setCheckboxTintList(..., defaultColorIdOfUncheckedState, defaultColorIdOfCheckedState);`
`setCheckboxTintList(..., defaultColorIdOfCheckedState, defaultColorIdOfUncheckedState);`

I guess I should rename those vars